.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
 env:
   matrix:
   - JOB=PYUNIT
+  - JOB=TBANS_TESTS
```

Comment:
Unless someone can help me get the tests running via the Python script that runs our tests :)